src/openforms/js/components/form/file.js

```diff
@@ -27,6 +27,32 @@ const REGISTRATION = {
       },
       valueProperty: 'url',
     },
+    {
+      type: 'textfield',
+      key: 'registration.bronorganisatie',
+      label: 'Bronorganisatie',
+      description: 'RSIN of the organization which creates the ZAAK.',
+    },
+    {
+      type: 'select',
+      key: 'registration.vertrouwelijkheidaanduiding',
+      label: 'Vertrouwelijkheidaanduiding',
+      widget: 'choicesjs',
+      tableView: true,
+      data: {
+        get values() {
+          return jsonScriptToVar('CONFIDENTIALITY_LEVELS');
+        },
+      },
+      description:
+        'Indication of the level to which extend the dossier of the ZAAK is meant to be public.',
+    },
+    {
+      type: 'textfield',
+      key: 'registration.titel',
+      label: 'Titel',
```

Comment:
```suggestion
      label: 'Title',
```
let's be consistent in our english source strings